packages/web/src/scss/components/Item/_theme.scss

```diff
@@ -1,5 +1,6 @@
-@use '@tokens' as tokens;
+@use '@global' as global-tokens;
+@use '../../settings/globals';
 
-$gap: tokens.$space-400;
-$icon-color-default: tokens.$action-selected-default;
-$icon-color-disabled: tokens.$action-selected-disabled;
+$gap: global-tokens.$space-500;
```

Comment:
There are different spacings in Figma:

- horizontal 700
- vertical 500
- gap between label and helper text 4 px

![image](https://github.com/user-attachments/assets/e7591f8b-f439-4197-a83f-c5f42c702881)